central/risk/manager/manager.go

```diff
@@ -241,14 +241,16 @@ func (e *managerImpl) reprocessImageComponentRisk(imageComponent *storage.Embedd
 
 	oldScore := e.imageComponentRanker.GetScoreForID(
 		scancomponent.ComponentID(imageComponent.GetName(), imageComponent.GetVersion(), os))
-	if err := e.riskStorage.UpsertRisk(riskReprocessorCtx, risk); err != nil {
-		log.Errorf("Error reprocessing risk for image component %s v%s: %v", imageComponent.GetName(), imageComponent.GetVersion(), err)
-	}
 
+	// Image risk results are not currently used so if the score is the same then no need to upsert
 	if oldScore == risk.GetScore() {
 		return
 	}
 
+	if err := e.riskStorage.UpsertRisk(riskReprocessorCtx, risk); err != nil {
+		log.Errorf("Error reprocessing risk for image component %s v%s: %v", imageComponent.GetName(), imageComponent.GetVersion(), err)
```

Comment:
We should not append `v` to the version. The end user will likely include it in their searches consequently the search will fail.

Reply:
Will remove